sorting_algorithms/insertion_sort.py

```diff
@@ -0,0 +1,24 @@
+"""Create an insertion sort algorithm."""
+
+
+def insertion_sort(input):
```

Comment:
Use a different variable name than `input`, since that's a function built in to Python